tests/filecheck/parser-printer/builtin_attrs.mlir

```diff
@@ -8,6 +8,9 @@
 
   // CHECK: (index)
 
+  "func.func"() ({}) {function_type = () -> (), sym_name = "unit_attr_func", unitarray = [unit]} : () -> ()
```

Comment:
Can I suggest using the `test` dialect here? I don't think we want to depend on `func` here...

Reply:
I agree in general, although here it seemed like a good fit for consistency, happy to change it if you or anyone else insist